src/field_theory/intermediate_field.lean

```diff
@@ -395,6 +398,9 @@ def restrict_scalars (E : intermediate_field L' L) :
 @[simp] lemma coe_restrict_scalars {E : intermediate_field L' L} :
   (restrict_scalars K E : set L) = (E : set L) := rfl
 
+@[simp] lemma coe_sort_restrict_scalars (E : intermediate_field L' L) :
+  ↥(restrict_scalars K E) = ↥E := rfl
```

Comment:
This is a bad lemma because it removes the `K`-action from `restrict_scalars K E`, the whole point of which was to add the `K` action.

This type of lemma feels a bit like making `mul_opposite A = A` a lemma.